pkg/lightning/restore/tidb.go

```diff
@@ -219,7 +219,7 @@ func createTableIfNotExistsStmt(p *parser.Parser, createTable, dbName, tblName s
 			return []string{}, err
 		}
 		ctx.WritePlain(";")
-		retStmts = append(retStmts, res.String())
+		retStmts = append(retStmts, binloginfo.AddSpecialComment(res.String()))
```

Comment:
👀 

why are we using a regexp-based solution (for CDC), it will break edge cases like

```sql
CREATE TABLE `PRIMARY KEY CLUSTERED` (
    `SECONDARY KEY CLUSTERED` VARCHAR(100) DEFAULT 'TERTIARY KEY CLUSTERED'
);
```